modules/quickstart-sap/pages/configure.adoc

```diff
@@ -19,27 +27,20 @@ If you perform these steps in the wrong order, your SAP installation will fail.
 ====
 
 
-To install the SAP formulas, you need the ``SLES for SAP`` product from {scc}.
-In the {productname} {webui}, navigate to menu:Admin[Setup Wizard > Products] and search the list for ``SUSE Linux Enterprise Server for SAP Applications 15 x86_64``.
-Select this parent product.
-The required child products are automatically selected.
-Click btn:[Add Products] and wait until the products have finished synchronizing.
-
-
-
-
 
 == Enable and Configure the HANA Formula
 
 In the {productname} {webui}, navigate to menu:Systems[System List] and click the client to use as the {productname} master.
 
-Navigate to the [guimenu]``Formulas`` tab and check the formula in the list.
+Navigate to the [guimenu]``Formulas`` tab and check the ``habootstrap`` formula in the list.
 Click btn:``Save`` and apply the highstate to save the changes.
```

Comment:
```suggestion
Click btn:``Save`` and apply the highstate to save the changes.
```
something wrong with the btn markup.
shouldn't you better say: "to install/deploy the changes"?